frontend/src/components/ContextMenu.js

```diff
@@ -0,0 +1,75 @@
+/*
+  Context menu shows a list of actions the user can select to perform actions. The actions
+  are hidden until the user opens the menu (the three dot icon). The menu is automatically
+  closed on blur. Pass this component an array of objects with the label and action to tie
+  to that label. Be sure to pass in a description of the menu in the `label` prop. This prop
+  is used as ellipsis' aria-label.
+*/
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEllipsisH } from '@fortawesome/free-solid-svg-icons';
+import { Button } from '@trussworks/react-uswds';
+
+import './ContextMenu.css';
+
+function ContextMenu({
+  label, menuItems, backgroundColor, left,
+}) {
+  const [shown, updateShown] = useState(false);
+  const defaultClass = 'smart-hub--context-menu';
+  const menuClass = left ? `${defaultClass} smart-hub--context-menu__left` : defaultClass;
+
+  const onBlur = (e) => {
+    const { currentTarget } = e;
+
+    setTimeout(() => {
+      if (!currentTarget.contains(document.activeElement) && shown) {
+        updateShown(false);
+      }
+    }, 0);
+  };
+
+  return (
+    <div onBlur={onBlur}>
+      <Button
+        type="button"
+        className="smart-hub--context-menu-button smart-hub--button__no-margin"
+        unstyled
+        onClick={() => updateShown((previous) => !previous)}
+        aria-label={label}
```

Comment:
Good use of the `aria-label` but I think for accessibility there's likely some more we need to do. [`aria-haspopup`](https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20161214/examples/menu-button/menu-button-1/menu-button-1.html) might be a good thing to look at

Reply:
Here's probably a better link for `aria-haspopup` https://www.w3.org/TR/wai-aria-1.1/#aria-haspopup